phovea_server/set_encoder.py

```diff
@@ -1,25 +1,21 @@
 """
-This encoder is required to handle changes of data types in Python 3.7 by decoding bytes objects to strings and adding list() to set().
+This encoder is required to handle changes of data types in Python 3.7 by adding list() to set().
 """
 
 
-class Python3Encoder(object):
+class SetEncoder(object):
   def __contains__(self, obj):
-    if isinstance(obj, bytes):
-      return True
     if isinstance(obj, set):
       return True
     return False
 
   def __call__(self, obj, base_encoder):
-    if isinstance(obj, bytes):
-      return obj.decode('utf-8')
     if isinstance(obj, set):
       return list(obj)
     return None
```

Comment:
Due to the way this Encoder is called (https://github.com/phovea/phovea_server/blob/develop/phovea_server/util.py#L24-L26), these lines are only called on actual `set` instances. Therefore, they cannot be `None` or any other instance. You can replace this by `return list(obj)` then.